homeassistant/components/sensor/airvisual.py

```diff
@@ -41,33 +41,39 @@
 SENSOR_TYPE_AQI = 'air_quality_index'
 SENSOR_TYPE_POLLUTANT = 'main_pollutant'
 SENSORS = [
-    (SENSOR_TYPE_LEVEL, 'Air Pollution Level', 'mdi:scale', None),
-    (SENSOR_TYPE_AQI, 'Air Quality Index', 'mdi:format-list-numbers', 'AQI'),
+    (SENSOR_TYPE_LEVEL, 'Air Pollution Level', 'mdi:gauge', None),
```

Comment:
Will the `mdi:gauge` icon ever actually appear (given that it's being overridden by the icons in `POLLUTANT_LEVEL_MAPPING`?

Reply:
It can only appear if AQI is either negative, or more than 10000. So, highly improbable, but technically possible. Just a habit of covering all the cases :-)

Reply:
Sweet! 👍